index.html

```diff
@@ -217,7 +217,316 @@ <h2 class="card-heading">Multi-Post Stories</h2>
+                        <li><a href="#">javascript</a></li>
+                        <li class="mode-text"><a href="#">Github</a></li>
+                        <li class="mode-text"><a href="#">Ruby</a></li>
+                        <li class="mode-text"><a href="#">Bootstrap</a></li>
+                      </ul>
+                      <ul class="modal-live modal-live-web">
+                        <li><a href="https://mikelobi.github.io/Personal-Portfolio/" class="button modal-button">See Live<img class="modal-icon" src="image/Icon.png" alt="live link icon"/></a></li>
+                        <li><a href="https://github.com/Mikelobi/Personal-Portfolio" class="button modal-button">See Source<img class="modal-icon" src="image/Vector.png" alt="github link icon"/></a></li>
+                      </ul>
+                    </div>
+                  </div>
+              </div>            
+            </div>
+            <div class="image-footer">
+              <img src="image/bar.png" alt="bottom bar" class="modal-work-footer-image" />
+            </div>
+          </div>
+          <div id="myModal7" class="modal-work-web">
+            <!-- Popup content -->
+            <div class="modal-work-content-web">
+              <div class="modal-content-web">
+                  <div class="modal-title">
+                    <h3>Tonic</h3>
+                    <span class="close-work7">&times;</span>
+                  </div>
+                  <div class="modal-tool">
+                    <ul>
+                      <li>CANOPY</li>
+                      <li><img src="image/Counter.png" alt="dot"></li>
+                      <li class="fint">Back End Dev</li>
+                      <li><img src="image/Counter.png" alt="dot"></li>
+                      <li class="fint">2015</li>
+                    </ul>      
+                  </div>
+                  <div class="modal-body">
+                    <img src="image/Snapshoot-Portfolio-4.png" alt="a baby" class="web">
+                    <img src="image/Snapshoot-Portfolio.png" alt="A photo of a website page" class="mob">
+                  </div>
+                  <div class="web-modal">
+                    <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent<span class="mode-text">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.</span></p>
+                    <div class="links-modal">
```

Comment:
- [ ] Please make sure this text matches the Figma template in terms of text color and font-family, font-size, for a better UI.